project/src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,92 @@
+package com.google.sps.data;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public final class User {
+  private final long userId;
+  private String name;
+  private String email;
+  private Set<String> interests;
+  private Set<String> skills;
+  private Set<Event> eventsHosting;
+  private Set<Event> eventsParticipating;
+  private Set<Event> eventsVolunteering;
+
+  /**
+   * Creates a new user.
+   *
+   * @param name The human-readable name for the user. Must be non-null.
+   * @param email The e-mail with which the user is reachable. Must be non-null.
+   */
+  public User(String name, String email) {
+    this.userId = 10000000L;
+    this.name = name;
+    this.email = email;
+    this.interests = new HashSet<>();
```

Comment:
I think you can move all these to line 10-14?

Reply:
Changed due to structure revamp